softwareconfiguration/autofe-reldata/src/main/java/autofe/db/model/relation/BackwardRelationship.java

```diff
@@ -0,0 +1,19 @@
+package autofe.db.model.relation;
+
+public class BackwardRelationship extends AbstractRelationship {
+
+	public BackwardRelationship() {
+		super();
+	}
+
+	public BackwardRelationship(String fromTableName, String toTableName, String commonAttributeName) {
+		super(fromTableName, toTableName, commonAttributeName);
+	}
+
+	@Override
+	public String toString() {
+		return "BackwardRelationship [fromTableName=" + fromTableName + ", toTableName=" + toTableName
+				+ ", commonAttributeName=" + commonAttributeName + "]";
```

Comment:
I think that code formatting here differs from our (line break)

Reply:
Resolved